src/components/seconardynavnew/tracking.tsx

```diff
@@ -0,0 +1,46 @@
+import { useEffect } from 'react';
+
+declare global {
+    interface Window {
+        segment: any;
+    }
+}
+
+export const useLinkTracking = (
+    el: HTMLElement | undefined | null,
+    callback: (e: Event) => void
+) => {
+    useEffect(() => {
+        el?.querySelectorAll('a').forEach((anchor: HTMLAnchorElement) => {
```

Comment:
I would have liked to use event bubbling here as this is a bit ugly, but unfortunately [flora hardcodes `e.stopPropagation()` in its link onClick handlers](https://github.com/10gen/flora/blob/a2f623775fb0719a4067cdc1c560b8fced48f648/src/Link/Link.tsx#L82).
As a result, I had to give a click listener to all child links manually via `querySelectorAll`. If anyone thinks of a better approach please let me know.